driver/examples/configs/test_external_C12_2x2.yaml

```diff
@@ -0,0 +1,102 @@
+stencil_config:
+  compilation_config:
+    backend: numpy
+    rebuild: false
+    validate_args: true
+    format_source: false
+    device_sync: false
+grid_config:
+  type: external
+  config:
+    grid_type: 0
+    grid_file_path: "../../test_input/C12.tile"
```

Comment:
See above.

Reply:
Since these are small files, they should be okay.  Ideally we'd store them in the container (mentioned above) or generate them on the fly to test the grid creation and ingest as a group.